subsys/fs/shell.c

```diff
@@ -33,6 +33,16 @@ static struct fs_mount_t fatfs_mnt = {
 /* LITTLEFS */
 #ifdef CONFIG_FILE_SYSTEM_LITTLEFS
 #include <zephyr/fs/littlefs.h>
+
+#ifdef CONFIG_FS_LITTLEFS_BLK_DEV
+FS_LITTLEFS_DECLARE_DEFAULT_CONFIG(lfs_data);
+static struct fs_mount_t littlefs_mnt = {
+	.type = FS_LITTLEFS,
+	.fs_data = &lfs_data,
+	.flags = FS_MOUNT_FLAG_USE_DISK_ACCESS,
+	.storage_dev = CONFIG_SDMMC_VOLUME_NAME,
```

Comment:
(note) it would be nice to have a support both of them at the same time, I mean flash and block devices
but I believe these changes will be enough for now

Reply:
Yes it is not for now